main.go

```diff
@@ -3,12 +3,14 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/openshift/ci-chat-bot/pkg"
```

Comment:
Other files have this renamed to `utils`, but this import using the package name of `util`. Either this should also be named `utils` or the other imports should use the package name. I think using the package name would be preferable.

Reply:
Done